src/exporters/index.ts

```diff
@@ -127,15 +129,10 @@ export default function exportTranslationKeys(
   }
 
   for (const [filePath, keysForFilepath] of Object.entries(keysPerFilepath)) {
-    if (!(filePath in cache.originalTranslationFiles)) {
-      // Cache original translation file so that we don't loose it across babel
-      // passes.
-      cache.originalTranslationFiles[filePath] = loadTranslationFile(
-        exporter,
-        config,
-        filePath,
-      );
-    }
+    cache.originalTranslationFiles[filePath] = deepmerge(
```

Comment:
Looks like deepmerge concatenates arrays by default. I don't think we export any arrays at the moment, but in the meantime I don't think we'll expect arrays to grow indefinitely if we export them at some point. Can you just pass `{ arrayMerge: (dest, source) => source }` as option to deepmerge?

Reply:
✅